bokehjs/src/coffee/core/selection_manager.ts

```diff
@@ -41,6 +41,11 @@ export class SelectionManager extends HasProps {
         }
       }
     }
+    return {'glyph_renderer_views': glyph_renderer_views, 'graph_renderer_views': graph_renderer_views}
```

Comment:
Drop quotes. This will signify that you work with interface members and not arbitrary keys.